### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by czh-rot** [ranran0523/SPECNN#1]
How soon will the code be released?

**Comment by czh-rot** [comment]
Thanks for the source code, but this seems to be just the implementation of NN, not yet observed how it combines with CKKS. Will you continue to improve this repo in the future?

**Comment by ranran0523** [comment]
Yes, the HE inference code has some problems with our server updates. I am debugging this right now.

**Comment by czh-rot** [comment]
> Yes, the HE inference code has some problems with our server updates. I am debugging this right now.



> Yes, the HE inference code has some problems with our server updates. I am debugging this right now.

Thanks for the reply and wish u the best

**Comment by czh-rot** [comment]
> Yes, the HE inference code has some problems with our server updates. I am debugging this right now.

Thanks for the reply and wish u the best!
